### PR TITLE
proxifier-portable: Update to version 4.07

### LINKED ISSUE #7577: Update proxifier-portable to version 4.07
## Bug Report

**Package Name:** proxifier-portable

### Current Behaviour

Versions of proxifier-portable is outdated at 3.42

### Expected Behaviour

Versions of proxifier-portable should be updated to 4.07

### Possible Solution

I will submit a PR to update the software version to 4.07

### System details

**Windows version:** 11

**OS architecture:** 64bit

**PowerShell version:** 7.2.1

### PR DESCRIPTION
**Pull request by Absolves**
proxifier-portable: Update to version 4.07

The latest version of proxifier-portable is 4.07 and in bucket is 3.42. So, I updated it to the latest version.
Closes #7577 

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
Add autoupdate as well

File: bucket/proxifier-portable.json [hunk 1]
```diff
@@ -23,6 +23,6 @@
     ],
     "checkver": {
         "url": "https://www.proxifier.com/download/",
-        "regex": "(?sm)Portable Edition v3.*changelog/win3.html\">([\\d.]+)</a>"
+        "regex": "(?sm)Portable Edition v4.*changelog/win3.html\">([\\d.]+)</a>"
     }
```
Comment (issaclin32) [inline bucket/proxifier-portable.json:27]:
```suggestion
    },
    "autoupdate": {
        "url": "https://www.proxifier.com/download/ProxifierPE.zip"
    }
```

**Pull request merged**